### DISCUSSION TITLE
这是NextChat的bug吗？

### DISCUSSION THREAD

**Discussion by Githubiee** [BurnHair/docs#158] [技术支持]
图片prompt用多了之后就变成这样
<img width="837" alt="iShot_2024-07-11_14 01 44" src="https://github.com/BurnHair/docs/assets/163315547/751319ad-2614-42bc-ab34-82b6d36e5e01">

**Comment by BennyThink** [comment]
没见过这个错误。你的额度用完了吗？ （如果不是删掉之前的对话就行了，浏览器存储满了。）
